nthprime/lib/nthprime.rb

```diff
@@ -0,0 +1,14 @@
+class Nthprime
+  attr_reader :n
+  def initialize(n)
+    @n = n
+  end
+
+  def primes
+    primes = [nil, nil, *2..n]
+    (2..Math.sqrt(n)).each do |i|
+      (i**2..n).step(i) { |j| primes[j] = nil } if primes[i]
```

Comment:
What is your reasoning for using n, i, and j as variable names? Is there a way to name these variables to be a little more readable? Does brevity trump readability (in terms of being able to read and immediately know what each variable stands for)?